salt/orch/update.sls

```diff
@@ -26,9 +26,17 @@ update_modules:
 
 # Get list of masters needing reboot
 {%- set masters = salt.saltutil.runner('mine.get', tgt='G@roles:kube-master and G@tx_update_reboot_needed:true', fun='network.ip_addrs', tgt_type='compound') %}
-
 {%- for master_id in masters.keys() %}
 
+# Ensure the node is marked as upgrading
+{{ master_id }}-set-update-grain:
+  salt.function:
+    - tgt: {{ master_id }}
+    - name: grains.setval
+    - arg:
+      - update_in_progress
```

Comment:
Nitpicking: Following the same `-` vs `_` policy, maybe this should also be in `-` instead of `_` ?